### PR TITLE
Fix cross-origin links are unsafe

### LINKED ISSUE #31528: [Issue] Fix cross-origin links are unsafe
This issue is automatically created based on existing pull request: magento/magento2#31460: Fix cross-origin links are unsafe

---------

### Description (*)

This fix for issue lighthouse audit fail related link missing rel attribute

### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### PR DESCRIPTION
**Pull request by mrtuvn**
### Description (*)

This fix for issue lighthouse audit fail related link missing rel attribute

### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### Resolved issues:
1. [x] resolves magento/magento2#31528: Fix cross-origin links are unsafe

### CODE REVIEW

**Comment by mrtuvn** [comment]
@magento run all tests

**Comment by mrtuvn** [comment]
@magento run Functional Tests CE, Functional Tests B2B, WebAPI Tests

**Comment by sidolov** [comment]
@magento create issue

File: app/code/Magento/Theme/view/frontend/templates/html/bugreport.phtml [hunk 1]
```diff
@@ -3,11 +3,15 @@
...
  */
+
+/**
+ * @var \Magento\Framework\Escaper $escaper
+ */
 ?>
 <small class="bugs">
-    <span><?= $block->escapeHtml(__('Help Us Keep Magento Healthy')) ?></span>
+    <span><?= $escaper->escapeHtml(__('Help Us Keep Magento Healthy')) ?></span>
     <a href="http://www.magentocommerce.com/bug-tracking"
```
Comment (ihor-sviziev) [inline app/code/Magento/Theme/view/frontend/templates/html/bugreport.phtml]: Maybe 
```suggestion
    <a href="https://www.magentocommerce.com/bug-tracking"
```

File: app/code/Magento/Theme/view/frontend/templates/html/footer.phtml [hunk 2]
```diff
@@ -3,14 +3,18 @@
...
+/**
+ * @var \Magento\Framework\Escaper $escaper
+ */
 ?>
 <div class="footer-container">
     <div class="footer">
         <?= $block->getChildHtml() ?>
-        <p class="bugs"><?= $block->escapeHtml(__('Help Us Keep Magento Healthy')) ?> - <a
+        <p class="bugs"><?= $escaper->escapeHtml(__('Help Us Keep Magento Healthy')) ?> - <a
             href="http://www.magentocommerce.com/bug-tracking"
```
Comment (ihor-sviziev) [inline app/code/Magento/Theme/view/frontend/templates/html/footer.phtml]:
```suggestion
            href="https://www.magentocommerce.com/bug-tracking"
```

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: app/code/Magento/Theme/view/frontend/templates/html/bugreport.phtml (hunk 1 above)
Comment (mrtuvn) [reply app/code/Magento/Theme/view/frontend/templates/html/bugreport.phtml]: Should i use github link issues with https instead ? https://www.magentocommerce.com seem weird and a bit confusing
We can use https://github.com/magento/magento2/issues

Comment (ihor-sviziev) [reply app/code/Magento/Theme/view/frontend/templates/html/bugreport.phtml]: @sidolov @sivaschenko, can we put here a link to the GitHub repository?

Comment (sidolov) [reply app/code/Magento/Theme/view/frontend/templates/html/bugreport.phtml]: @mrtuvn , @ihor-sviziev yes, we can use link to GitHub

**Comment by mrtuvn** [comment]
Updated my PR as suggestions

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests CE

**Review by ihor-sviziev — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-8623](https://jira.corp.magento.com/browse/ENGCOM-8623) has been created to process this Pull Request

**Comment by engcom-Bravo** [comment]
Hi @mrtuvn,

We are closing this PR.The PR is in Draft since longtime.

Once you are ready Please feel free to reopen it to process the PR further.

Thanks

**Pull request closed without merging**